### ISSUE TITLE
about code

### ISSUE THREAD

**Issue by SaintLogos1234** [leeyeehoo/SiamVGG#5]
Thank you very much for your work. May I ask when the complete code will be released

**Comment by leeyeehoo** [comment]
soon

**Comment by SaintLogos1234** [comment]
I see that you have published the training code. Could you please update the readme and explain your training steps in detail

**Comment by SaintLogos1234** [comment]
I read yor code, I can not find rpn module

**Comment by SaintLogos1234** [comment]
I have another question， I only use Ilsvrc dataset to train， don't use  youtube, does it have a big impact on the performance of the model?Finally, how long did it take you to train

**Comment by SaintLogos1234** [comment]
Last question, could you please tell me where your youtube dataset was downloaded? I didn't see the download link on its official website

**Comment by SaintLogos1234** [comment]
/home/xieyangyang/桌面/选区_005.bmp，这个深度应该是256吧

**Comment by leeyeehoo** [comment]
It will impact the accuracy but not too much.  You can follow https://github.com/mbuckler/youtube-bb to download and process the dataset.

**Comment by leeyeehoo** [comment]
The rpn module is  in https://github.com/leeyeehoo/SiamRPN-VGG 

I dint' finish the readme of both sorry for that

**Comment by SaintLogos1234** [comment]
For some reason, I can't download the youtube-bb dataset in your way, Because of certain government regulations, we can't connect directly to the outside world， By the way, do you have any other download method recommended

**Comment by leeyeehoo** [comment]
I'm really sorry about that. Because it is a huge dataset, we cannot upload it to Baidu Cloud or else. You may try VPN to download it.
